23.md

```diff
@@ -0,0 +1,73 @@
+LUD-23: Webhook notification for a third-party in pay protocol.
+=================================
+
+`author: hsjoberg` `author: andreneves` `author: fiatjaf`
+
+---
+
+## Third-party webhook notification
+
+In cases where an LNURL-pay code is displayed on a third-party website,
+it could make sense to be able to give feedback to the user upon a payment
+completion.
+
+## Construction
+
+In order to accomplish this, the third party can re-construct the bech32 code or
+build a [LUD-17](./17.md) raw LNURL code, with a `webhook` query parameter
+amended to the URL. The value of `webhook` is a URL that should be called by the
+wallet upon payment completion.
```

Comment:
Why is wallet calling the webhook? I thought service is calling the webhook upon invoice settlement.

Reply:
Oh it's not supposed to, the sentence is erroneous. Thanks.